client/src/Tools/_framework/Paths/Activities.jsx

```diff
@@ -354,6 +371,21 @@ export function Activities() {
         content={folder.content}
         allDoenetmlVersions={allDoenetmlVersions}
       />
+      {folderId ? (
```

Comment:
If you move this whole thing after the heading and change the style to
```
            style={{
              marginLeft: "15px",
              marginTop: "-40px",
              float: "left",
            }}
```
then it doesn't push the heading down and it doesn't shift the headng left-or right. I don't know if it is OK to use a negative margin like that, but it seems to work on at least one web browser. (I'm not too well versed in css so don't know if negative margins are frowned on or acceptable.)